### PR TITLE
Upgrade to latest swift

### PR DESCRIPTION
**Pull request by dtweston**
This PR upgrades this project to the latest Swift syntax.

### CODE REVIEW

File: AXSwift/Observer.swift [hunk 1]
```diff
@@ -104,32 +104,36 @@ public final class Observer {
...
+private func internalCallback(_ axObserver: AXObserver,
                               axElement: AXUIElement,
                               notification: CFString,
-                              userData: UnsafeMutablePointer<Void>) {
-  let observer = Unmanaged<Observer>.fromOpaque(COpaquePointer(userData)).takeUnretainedValue()
-  let element  = UIElement(axElement)
-  let notif    = Notification(rawValue: notification as String)!
-  observer.callback!(observer: observer, element: element, notification: notif)
+                              userData: UnsafeMutableRawPointer?) {
+  if let userData = userData {
```
Comment (tmandry) [inline AXSwift/Observer.swift]: I can't think of any reason why this should be nil, can you? It looks like the old code assumed it was present (and nil would indicate a bug somewhere). I would rather not mask potential bugs if that's still the case.

Comment (dtweston) [inline AXSwift/Observer.swift]: I can't think of any reason. How about I replace the `if let` with:
````
  guard let userData = userData else { fatalError("userData should be an AXSwift.Observer") }
````

**Comment by tmandry** [comment]
Yep that sounds good.

**Comment by dtweston** [comment]
Ok, I pushed the `guard let` change, and added a commit that modifies the indentation settings at the project level (since they're different from the Apple standard)

**Comment by tmandry** [comment]
Merged. Thanks!

**Comment by dtweston** [comment]
No problem! I can't wait to use it in my next project. :)

**Comment by tmandry** [comment]
Mind if I ask what you're using it for?

**Comment by dtweston** [comment]
Heh. I was thinking about using it to build a collaborative playground editor. But, with WWDC right around the corner, I realize that I should wait to see what new things Apple is going to provide first. :)

**Pull request merged**